### DISCUSSION TITLE
Converting Songs to Extended Version

### DISCUSSION THREAD

**Discussion by hnnst** [sigma67/spotify_to_ytmusic#98] [Ideas]
Hello Developers!
I would really like to transfer songs from my Spotify Playlists to a YouTube playlist, but instead of the original songs i would like to get an extended version of the song from YouTube. So for example the song "XYZ" in the Spotify Playlist will be "XYZ (Extended Version)" in the resulting YouTube Playlist.

**Comment by sigma67** [comment]
Not really possible unfortunately

The matching is heavily based on the duration of the song, which ensures the good quality matches
